fladgejt/webui/predmety.py

```diff
@@ -51,11 +54,11 @@ def get_informacny_list(self, kod_predmetu, akademicky_rok):
         assert_ops(ops, 'closeDialog', 'abortBox')
         app.close_dialog(*ops[0].args)
         with app.collect_operations() as ops:
-            app.abort_box()
+            app.abort_box('')
 
-        # Otvori sa vysledne PDF.
+        # Vratime zakodove PDF.
         url = app.awaited_shell_exec(ops)
-        return app.context.request_text(url)
+        return base64.b64encode(url.content).decode()
```

Comment:
Hmm. Príliš sa mi nepáči posielať klientovi celé PDF ako base64. Omnoho lepšie by bolo poslať mu len url, ktorú by proste dal do href. Ale to by vyžadovalo relatívne veľké zmeny...
- shell_exec a awaited_shell_exec nebudú sťahovať response, iba vrátia url.
- vrátená url bude začínať na `self.context.ais_url`.
- bohužiaľ treba upraviť aj v ankete `scripts/votr_export.py`, ktorý teraz volá awaited_shell_exec a očakáva response, aby sám zavolal `response = app.context.request_ais(url)`. Čo obnáša update Votru v anketa dependencies... ech...

Ale asi to preháňam. Možno je lepšie mať aspoň niečo, a optimalizovať sa to dá potom?